README.md

```diff
@@ -70,11 +70,11 @@ You should now be able to run Cylc commands, e.g.:
 Start the Hub (JupyterHub gets installed with the "cylc" package):
 
 ```sh
-mkdir -p "${HOME}/srv/cylc/"
+mkdir -p "${HOME}/srv/cylc/"  # the hub will store session information here
 cd "${HOME}/srv/cylc/"
 jupyterhub \
     --JupyterHub.spawner_class="jupyterhub.spawner.LocalProcessSpawner" \
-    --JupyterHub.logo_file="${CONDA_PREFIX}/work/cylc-ui/img/logo.png" \
+    --JupyterHub.logo_file="${CONDA_PREFIX}/work/cylc-ui/img/logo.svg" \
```

Comment:
Oh, I thought we didn't have a logo file! Thanks for fixing that (/me throws the post-it saying "Fix conda cylc logo" in the trash bin)

Reply:
We should use the coloured one in this context, but greyscale is fine for the moment.